src/App.jsx

```diff
@@ -1,9 +1,11 @@
-function App() {
+import JokeRender from "./components/exercise8/jokeRender";
 
+function App() {
   return (
     <>
+      <JokeRender />
     </>
```

Comment:
```suggestion
      <JokeRender />
```
I believe there is no need for this fragment here.

Reply:
Thank you for pointing that out